frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/ability_to_filter_quick_starts_catalog_ODC-4872.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Add ability to filter Quick Starts catalog
+	As a user, i want to be able to filter Quick Starts in the catalog by status of the Quick Starts and search Quick Starts catalog by name, description, tags and prerequisites.
+
+Background:
+    Given user is at developer perspective
+    And user has created CR for sample-application Quick Start
+    And user has created CR for install-associate-pipeline Quick Start
+    And user has created CR for explore-pipeline Quick Start
+    And user has created CR for add-healthchecks Quick Start
+
```

Comment:
User doesn't need to create quick start CRs always. Default quick starts will be created by console operator automatically.

Reply:
`install-associate-pipeline ` will be delivered by pipelines operator.

Reply:
This is just the given statements if there are already there then it is fine too

Reply:
I've updated `install-associate-pipeline` with `explore serverless`